docs/datasets.md

```diff
@@ -9,7 +9,7 @@ tfds.list_builders()
 
 # Load a given dataset by name, along with the DatasetInfo
 data, info = tfds.load("mnist", with_info=True)
-train_data, test_data = data['test'], data['train']
+train_data, test_data = data['train'], data['test']
```

Comment:
This file is generated automatically using: "scripts/document_datasets.py".

Please modify the code there.

Reply:
Sure, I wasn't aware that these are generated docs. Updated. :+1: